django/db/backends/base/operations.py

```diff
@@ -177,16 +177,26 @@ def force_no_ordering(self):
         """
         return []
 
-    def for_update_sql(self, nowait=False, skip_locked=False):
+    def for_update_sql(self, nowait=False, skip_locked=False, of=[]):
```

Comment:
I never think mutable keyword arguments are a good idea, even in this case where they will never be mutated. I am surprised `flake8` didn't pick this up.